public/javascripts/Gallery/src/keyboard/Keyboard.js

```diff
@@ -1,9 +1,9 @@
 /**
  * A Keyboard module.
- * @param {Modal} modal The object for the expanded view modal in the gallery
+ * @param {ExpandedView} ExpandedView The object for the expanded view modal in the gallery
  * @constructor
  */
-function Keyboard(modal) {
+function Keyboard(ExpandedView) {
```

Comment:
Oh, didn't notice this last time: naming convention for parameters is camelCase, not PascalCase